all.yml

```diff
@@ -1,6 +1,7 @@
 aws_ami:
 aws_autoscaling_group:
 aws_cloudformation_stack:
+aws_cloudtrail_trail:
```

Comment:
```suggestion
aws_cloudtrail:
```

Reply:
Done and done; thank you for the pointer!